src/canPDC.cpp

```diff
@@ -0,0 +1,21 @@
+#include "canPDC.h"
+
+CANPDC::CANPDC(CAN_TypeDef* canPort, CAN_PINS pins, int frequency) : CANManager(canPort, pins, frequency) {};
+
+void CANPDC::readHandler(CAN_message_t msg) {
+    
+}
+
+void CANPDC::sendPDCData() {
+    readAnalog();
```

Comment:
These read calls aren't necessary because the TimerInterrupt will be polling and updating the values. There is an argument that, if these values were only used for sending data via CAN, that having the read calls in here and removing the TimerInterrupt will be the best move since we don't need to poll more than we send. However, the state machine for the motor controller will be added to this repo soon, and that will require the TimerInterrupt for the values. 

For the time being, let's remove these read calls.